kivy/uix/actionbar.py

```diff
@@ -249,6 +249,10 @@ class ActionGroup(ActionItem, Spinner):
        defaults to 'normal'.
     '''
 
+    dropdown_width = NumericProperty(0)
+    '''If non zero, provides the width for the associated DropDown.
```

Comment:
`versionadded` should come after the property/default descriptions, but both changes are required before this PR can be merged.